julia/oxygen/Project.toml

```
@@ -0,0 +1,5 @@
+[deps]
+Oxygen = "df9a0d86-3283-4920-82dc-4555fc0d1d8b"
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
+[compat]
+Oxygen = "^1.4.0"
```

```suggestion
Oxygen = "^1.5.0"
```
or

```suggestion
Oxygen = "~1.5.0"
```


Not sure which one guarantees to use the 1.5.x version and is more standard in the Julia community

---

I'd recommend dropping the compat section or at least changing it to `Oxygen = "^1"` so it grabs the latest 1.x.x version

---

[19cb81c](https://github.com/the-benchmarker/web-frameworks/pull/7286/commits/19cb81c4ae75855de69891e5069c9ed36b8d9a5f)